stress/src/throughput.rs

```diff
@@ -15,37 +22,65 @@ where
         STOP.store(true, Ordering::SeqCst);
     })
     .expect("Error setting Ctrl-C handler");
-    let mut start_time = Instant::now();
-    let mut end_time = Instant::now();
-    let mut total_count_old: u64 = 0;
-    let total_count = Arc::new(AtomicU64::new(0));
-    let total_count_clone = Arc::clone(&total_count);
-
-    rayon::spawn(move || {
-        (0..num_cpus::get()).into_par_iter().for_each(|_| loop {
-            func();
-            total_count_clone.fetch_add(1, Ordering::SeqCst);
+    let num_threads = num_cpus::get_physical();
+    println!("Number threads: {}", num_threads);
```

Comment:
Nit - I understand this is stress test, and so would like to utilize all the cores. But as the enhancement separately, if we can also make num_threads configurable (optionally passed as command line argument, and use total available cores otherwise), it will help test how the metrics SDK scales as we increase number of concurrent requests.

Reply:
Can leave this for a seperate pr.